### ISSUE TITLE
Throw errors for unhandled requests without providing a mock

### ISSUE THREAD

**Issue by harmdewit** [wheresrhys/fetch-mock#428]
To disallow all requests, without those being stubbed it would be nice to be able to "initialize" fetchMock without having to mock an actual url.

This doesn't raise an error because nothing hasn't been "initialized" yet:
```javascript
const res = await fetch('http://example.com');
assert(res.ok);
```

This does raise an error `Error: fetch-mock: No fallback response defined for GET to http://example.com/`:

```javascript
fetchMock.mock('someunknownurl', 200);
const res = await fetch('http://example.com');
assert(res.ok);
```

**Comment by wheresrhys** [comment]
The behaviour you're after does exist as an internal, undocumented method `._mock()`.

I'd accept a PR that either
- aliases this to an `init()` method (or happy to discuss alternative names to this)
- allows `mock()` to be called with no arguments/`null` as its only argument, which skips the adding a route bit of `.mock()` and just skips straight to setting up a mock function

Not sure which option I prefer.

Please include at least one test and documentation updates in any PR.

**Comment by wheresrhys** [comment]
Released as 7.6.0
